docs/modules/ROOT/pages/running-commodore.adoc

```diff
@@ -45,7 +45,47 @@ commodore() {
 == macOS X
 
 On macOS with Docker for Mac mounting the SSH agents socket into a container doesn't work.
-Instead your SSH key needs to be mounted into the container.
+Instead you need to mount as magic path that's provided by Docker for Mac.
+
+[NOTE]
+====
+This only works for the stock ssh-agent coming along with macOS.
+If you use any other agent, you are out of luck.
+Docker for mac doesn't support mounting sockets.
+====
+
+[CAUTION]
+====
+That magic socket path belongs to root.
+One must a container with `--user=0` in order to access it.
```

Comment:
there seems to be a word missing in `One must [..] a container`.
Btw, "One must ..." is very formal English. You could also just write "You have to [start?] a container..."

Reply:
Yes the word _run_ was missing.